ironfish-cli/src/commands/miners/pools/status.ts

```diff
@@ -109,7 +109,10 @@ export class PoolStatus extends IronfishCommand {
 
     if (status.addressStatus) {
       result += `\nMining status for address '${status.addressStatus.publicAddress}':\n`
-      result += `Miners:                ${status.addressStatus.miners}\n`
+      result += `Number of miners:      ${status.addressStatus.miners}\n`
+      result += `Connected miners:      ${JSON.stringify(
+        status.addressStatus.connectedMiners,
+      )}\n`
```

Comment:
You shouldn't serialize this back to JSON, just use this to format this array of strings. status.addressStatus.connectedMiners.join(', ')